### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project I did:

- [ ] Used grid items
- [ ] Used flexbox property
- [ ] Added responsiveness

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,13 +43,19 @@ <h2>I'm a Software Developer</h2>
...
-        </div>        
+        </div> 
+        <img src="images/Illustration-header.png" alt="">       
     </section>
     <section class="works">
-        <h2 class="card-1">My Recent works</h2>
+        <div class="card-1">
+            <h2 >My Recent works</h2>
+            <img src="images/line.png" alt="line">
+        </div>
```
Comment (mahbubul14) [inline index.html:53]: - [ ] The featured project image should take 60% of the page's width and is aligned to the left, the description takes the remaining 40% and is aligned to the right. :+1: 

- [ ] The `See Project` button should take you to the Live Demo of the featured project or to the GitHub repository. :+1: 

- [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
works1_design | works1_figma

File: index.html [hunk 2]
```diff
@@ -129,22 +146,24 @@ <h2>Profesional Art Printing Data</h2>
...
-        <a class="see-project button-1" href="#">Get my resume</a>
-        <img src="images/Frame.png" alt="frame image">
-        <hr>
+        <div id="about-info">
+            <h2>About me</h2>
+            <p>
+                Hello I'm a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
```
Comment (mahbubul14) [inline index.html:159]: - [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
about_design | about_figma

File: index.html [hunk 3]
```diff
@@ -155,33 +174,32 @@ <h2>About me</h2>
...
-                        <li>Version Control</li>
-                        <li>CLI</li>
-                        <li>API Design</li>
-                        <li>Web development</li>
-                    </ul>
-                </li>
-            </ul>
         </div>
     </section>
     <section class="contact-form">
```
Comment (mahbubul14) [inline index.html:205]: - [ ] You should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Here, you should increase margin on the left side and `Get in touch` button should move to a little bit left to align with other text on the form and also give space under the social media icon. Kindly work on it. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
contact_design | contact_figma

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @tumainimaganiko,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by tumainimaganiko** [comment]
Hopefully, I've updated the required changes respectfully

**Review by topeogunleye — APPROVED** [review]
### Hi @tumainimaganiko, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
-  Featured project image takes 60% of the page's width ✔️ 
- See Project button links to the Live Demo of the GitHub repository. 👍
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**